client/src/clock-register/clock-register.controller.js

```diff
@@ -44,6 +44,13 @@
                 break;
               }
 
+              case 500: {
+                if (error.data.message) {
```

Comment:
Absolutely. We completely forgot about that `if` in the beginning of `catch` callback. As far as I remember, we started discussing about "error 500" and using a "convention" to only deal with more specific body formats (like objects that contains a certain key), then we did this. But I agree with you that we should be more concise.